cli/operator/node.go

```diff
@@ -92,6 +94,26 @@ var globalArgs global_config.Args
 
 var operatorNode operator.Node
 
+type ForkingMessageValidation struct {
```

Comment:
I wouldn't probably leave this comment for another file, but I'd extract this into another file (even in the same package is fine) because `node.go` is already quite big and complex to navigate, although I understand this is temporary